### PR TITLE
Remove references to Profiling insights

### PR DESCRIPTION
**Pull request by jfree**
### What does this PR do? What is the motivation?
Removes references to Watchdog insights for profiling since these have been removed from the IDE extensions and plugins.  Also replaces a couple of screenshots that were showing this type of insight.

### Merge instructions

- [x ] Please merge after reviewing

### CODE REVIEW

**Comment by michaelcretzman** [comment]
@jfree we don't delete images because our translation lags a little bit. We have an automated process that runs and removes unused images.

Can you please:

1. put the old images back.
2. rename the new images with new names and update the links to them with the new names.

Then I can merge this PR.

Thanks

File: content/en/developers/ide_plugins/visual_studio/_index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The Datadog extension for Visual Studio helps you find and fix bugs, security is
 
 ### Code insights
 
-Stay informed about [Error Tracking][5] issues, [Security Vulnerabilities][6], [Flaky Tests][10], and [Watchdog][7] profiling insights without leaving Visual Studio.
+Stay informed about [Error Tracking][5] issues, [Security Vulnerabilities][6], and [Flaky Tests][10], and without leaving Visual Studio.
```
Comment (cswatt) [inline content/en/developers/ide_plugins/visual_studio/_index.md:33]:
```suggestion
Stay informed about [Error Tracking][5] issues, [Security Vulnerabilities][6], and [Flaky Tests][10] without leaving Visual Studio.
```

**Comment by jfree** [comment]
@michaelcretzman I updated, let me know if there is anything else you need.

**Comment by jfree** [comment]
@cswatt let me know if there is anything you need from me on this PR, thanks!

**Review by cswatt — APPROVED** [review]

**Pull request merged**